usaspending_api/local_settings.example.py

```diff
@@ -0,0 +1,9 @@
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'usaspending_api',
+        'USER': 'usaspending_api',
+        'PASSWORD': '',
+    }
+}
```

Comment:
yeah I had the vague feeling as I was writing it that it was going to date me. I'll work on updating this for this PR.